src/sgame/sg_buildable.cpp

```diff
@@ -35,6 +35,11 @@ static Cvar::Cvar<bool> g_indestructibleBuildables(
 		"g_indestructibleBuildables",
 		"make buildables impossible to destroy", Cvar::NONE, false);
 
+static Cvar::Range<Cvar::Cvar<float>> g_BPMinerQueueFraction(
+		"g_BPMinerQueueFraction",
+		"upon miner construction, this fraction of the miner max yield is queued",
+		Cvar::NONE, 0.5, 0, 1);
```

Comment:
I think we should set this to zero for now until it gets some testing.

I think we need a means to change gameplay changes.

Reply:
Well, one way to test is to merge it and then play on nightly or other servers that are rebasing frequently. But if some server owners agree to host it, that would work too.

I don't think we want to have a separate server for each gameplay change, especially for minor tweaks like this. It's like how on a major website they'll have 1000 A/B tests running at a time, and a given user might be in 100 of the experiment arms.

Reply:
> Well, one way to test is to merge it and then play on nightly or other servers that are rebasing frequently. But if some server owners agree to host it, that would work too.

It's interesting to read that from someone not maintaining a server.
I disagree on the idea that changes should be done silently, not on the actual change. Now, I'm one of the least active mod hosters. I think @sweet235  and @cu-kai  should be the ones talking on that topic.

Still, I'd like to remind you that upstream tend to refuse PRs that do change things, and tend to only accept opt-in, and often requires changes that might introduce bugs in code that's working perfectly in prod since months.
To me, and I really talk only about me here, it really feels like a 2 weights, 2 measures thing.

Reply:
To be honest, I get the impression this "changes that are opt-in" thing is very recent. I have the impression it started one or two months ago. And I'm not sure I like it to be honest. On one hand I would rather have non-broken gameplay for everyone and not just the servers which would have tweaked just the right number of "crutch" cvars even if opt-in changes can mean things are easier to get merged.

Reply:
I'll be glad to host any gameplay experiments. We need more of these if you ask me.

Reply:
>  have the impression it started one or two months ago. And I'm not sure I like it to be honest.

That's the only way people found to contribute without ending in endless political, fruitless discussions, is what I've been told in-game.